website/content/docs/enterprise/audit/exclusion.mdx

```diff
@@ -0,0 +1,285 @@
+your auditing process.
+
+<Warning title="Proceed with caution">
+
+  Excluding audit entry fields is an advanced feature. Use of exclusion settings
+  could lead to missing data in your audit logs.
+
+  **Always** test your audit configuration in a non-production environment
+  before deploying exclusions to production. And make sure to read the
+  [Vault security model](/vault/docs/internals/security) and
+  [filtering overview](/vault/docs/concepts/filtering) to familiarize yourself
+  with Vault auditing and filtering basics before enabling audit devices which use
+  exclusions.
+
+</Warning>
+
+Once you enable an audit device with exclusions, every audit entry Vault sends to
+that audit device is compared to an (optional) predicate expression (condition).
+Before being written to the audit log for the device, audit entries that match the
+expression are then modified to remove the fields specified for that condition.
+It is possible to specify multiple sets of condition and field combinations for an
+individual audit device.
+
+When you enable audit devices that use exclusion, the behavior of any existing audit
+device and the behavior of new audit devices that **do not** use exclusion remains
+unchanged.
+
+## `exclude` option
+
+The value provided with the `exclude` option must be a parsable JSON array (i.e. JSON or
+an escaped JSON string) of exclusion objects.
+
+### Exclusion object
+
+- `condition` `(string: <optional>)` - predicate expression using [filtering syntax](/vault/docs/concepts/filtering),
+when matched Vault will attempt to remove values identified by `fields`.
+- `fields` `(string[] <required>)` - collection of fields in the audit entry to exclude,
+identified using [JSON pointer](https://tools.ietf.org/html/rfc6901) syntax.
+
+```json
+[
+  {
+    "condition": "",
+    "fields": [ "" ]
+  }
+]
+```
+
+Each `condition` is compared against the same golden source, immutable audit entry.
+This means excluding a particular field in a preceding exclusion, won't impact a
+subsequent condition evaluation.
+
+### Exclusion examples
+
+#### Exclude response data (when present)
+
+This example does not supply a condition which means the specified fields will be
+excluded from any audit entry that contains them.
+
+```json
+[
+  {
+    "fields": [ "/response/data" ]
+  }
+]
+```
+
+#### Exclude request data (when present) for transit mounts
+
+This example uses a condition to restrict exclusion of the request data field only to
+audit entries with a mount type of `transit`.
+
+```json
+[
+  {
+    "condition": "\"/request/mount_type\" == transit",
+    "fields": [ "/request/data" ]
+  }
+]
+```
+#### Multiple exclusions
+
+This example uses multiple exclusions to exclude:
+
+* `data` from both the request and response when the mount type is `transit`.
+* `entity_id` from requests where the `/auth/client_token` in the entry matches `hmac` and then more than one other character.
```

Comment:
```suggestion
Use multiple JSON blocks to exclude:

* `data` from both the request and response when the mount type is `transit`.
* `entity_id` from requests where the `/auth/client_token` starts with `hmac`
  followed by at least one other character.
```

It's worth clarifying if `\"hmac.+\" only matches to strings that *start* with "hmac" or any string that contains "hmac" with at least one character after it. I'm assuming the former in the edit, but feel free to adjust the suggestion if it's the latter.